build/NuGetAdditionalFiles/RunCsc.cmd

```diff
@@ -1,2 +1,7 @@
-echo off
-dotnet %~dp0\csc.dll %*
+@echo off
+if defined DOTNET_HOST_PATH (
```

Comment:
You mentioned you researched CMD defined variables a bit. Did you come to have a preference between the following two styles we use?

``` cmd
if "%DOTNET_HOST_PATH%" != "" (
if defined DOTNET_HOST_PATH (
```